qabelbox/src/main/java/de/qabel/qabelbox/box/dto/BoxPath.kt

```diff
@@ -42,5 +41,8 @@ sealed class BoxPath() {
 
 
     fun toList(): List<String> = if (this is Root) emptyList() else parent.toList() + name
+
+    fun toReadable(): String = "/" + toList().joinToString("/")
```

Comment:
Yes i saw it, but prefer to create an extra PR to replace all box classes moved to core